mw-webapp/src/component/image/Image.cy.tsx

```diff
@@ -1,9 +1,13 @@
 import logo from "src/assets/mastersWayLogo.svg";
 import {LOGO_TEXT} from "src/component/header/Header";
 import {Image} from "src/component/image/Image";
+import {MODAL_CY} from "src/component/modal/Modal.cy";
 import {getDataCy} from "src/utils/cyTesting/getDataCy";
 
-const IMAGE_CY = "image";
+const IMAGE_CY = {
+  dataCy: "image",
```

Comment:
Did not found dataCy prop in CyDataImage interface. Am I wrong?

Reply:
Should be something like this 

const Image_CY = {
  dataCyTrigger: "trigger",
  dataCyContent: {
    dataCyOverlay: "overlay",
    dataCyClose: "close",
    dataCyContent: "content",
    dataCyOK: "confirm",
  },
  dataCyImage: "image",
};